### ISSUE TITLE
Chronograf "dropping" points with same timestamp 

### ISSUE THREAD

**Issue by Anaisdg** [influxdata/chronograf#4825]
Using: InfluxDB v1.7.1 and Chronograf v1.7.3

Data points for tag values  "california" and "texas" have the same timestamp.  

Running this query: 
``` 
from(bucket: "Weather")
 |>  range(start: dashboardTime)
 |> group(by: ["season"])
```
<img width="1433" alt="screen shot 2018-11-19 at 2 29 13 pm" src="https://user-images.githubusercontent.com/30506042/48733678-f8a11580-ec08-11e8-9fa7-0bac2b50fa2e.png">

Expecting: 2x as many data points in lines. Granted, the line would look very bad/weird with vertical lines since points share the same timestamp.  

Actual: Only showing half of the data points, specifically for the "texas" tag value. 

<img width="1431" alt="screen shot 2018-11-19 at 2 28 51 pm" src="https://user-images.githubusercontent.com/30506042/48733854-68af9b80-ec09-11e8-8589-937118d8cd46.png">

In other words, the returned vis looks like this query:
```
from(bucket: "Weather")
  |> range(start: dashboardTime)
  |> filter(fn: (r) => r.location == "texas")
```
<img width="1328" alt="screen shot 2018-11-19 at 2 45 15 pm" src="https://user-images.githubusercontent.com/30506042/48734025-d9ef4e80-ec09-11e8-9aa2-1bd473272a69.png">

**Comment by mattgagliardi** [comment]
I'm experiencing similar behavior with Chronograf 1.7.10.  Basically I'm doing a nested query to determine the largest disks (grouped by disk), then doing a select for the top 10 disks by size on that initial query.  The query delivers the desired results when I log into InfluxDB and run it there.  Grafana also displays the correct output in a table (FWIW the 10 results all have the same timestamp, which I expected...I just hide that column).

Within Chronograf the same query is resulting in only a single row in the table, the result/row displayed corresponds to the value I used in the TOP function.  If I do TOP("last",5) I get the 5th row/5th largest disk and so on.

The query I'm using is:
SELECT disk,TOP("last",10) AS "USED SPACE" FROM (SELECT last("used_latest") FROM "<DB>"."<retention_policy>"."disk" WHERE time > :dashboardTime: GROUP BY disk FILL(null))

TIA!

**Comment by afausti** [comment]
@russorat  We are seeing this same issue in Chronograf 1.7.9

While the InlfuxDB HTTP API returns all points the table display in Chronograf drop points with the same timestamp and showing only the last entry.

```
[['2018-11-02T14:16:49Z', 1965, '1453', 'r', 'validation_data_cfht'],
 ['2018-11-02T14:16:49Z', 1966, '1453', 'HSC-I', 'validation_data_hsc'],
 ['2018-11-02T14:16:49Z', 1967, '1453', 'HSC-R', 'validation_data_hsc'],
 ['2018-11-02T14:16:49Z', 1968, '1453', 'HSC-Y', 'validation_data_hsc']]
```
I think we need to reopen this.

**Comment by russorat** [comment]
bump

**Comment by Michael-J-Ward** [comment]
Ditto.

**Comment by sranka** [comment]
I wrote some test data by 5-times running a modified [influxdb-1.8.ts](https://github.com/influxdata/influxdb-client-js/blob/master/examples/influxdb-1.8.ts) example:
```
const date = new Date()
for (let i = 0; i < 6; i++) {
  const point1 = new Point('temperature')
    .tag('example', 'write' + i)
    .floatField('value', 10 + i * 5 + Math.random() * 5 * (i + 1))
    .timestamp(date)
  writeApi.writePoint(point1)
}
```
The flux script 
```
from(bucket: "my-bucket")
  |> range(start: dashboardTime)
  |> filter(fn: (r) => r._measurement == "temperature")
  |> group(columns: ["example"])
```
then returns data that is rendered by the LineChart as expected:
```
![image](https://user-images.githubusercontent.com/16321466/84103871-82474c80-aa14-11ea-8970-3093313ccc81.png)
```
Feel free to reopen, if there are different steps to reproduce this issue. Having the same timestamp does not seem to be an issue anymore.

**Comment by desa** [comment]
@danatinflux are you still seeing this in 1.8.6?

**Comment by danatinflux** [comment]
I believe it's fixed in 1.8.6:

![image](https://user-images.githubusercontent.com/44617130/92029633-b0d67a00-ed1a-11ea-878c-bd40ba352940.png)
